joker/ghost_cola.lua

```diff
@@ -0,0 +1,48 @@
+SMODS.Joker {
+    key = 'ghost_cola',
+    loc_txt = {
+        name = "Ghost Cola",
+        text = {
+            "On {C:attention}Sell{}, create a {C:dark_edition}Negative{} Tag",
+            "and a random {C:dark_edition}Negative{} {C:spectral}Spectral{} card.",
```

Comment:
Adapting the description from Diet Cola:

"Sell this card to create a ~~free~~ {orange}Negative Tag{}
and a random {dark}Negative{} {spectral}Spectral{} card."

Something like that maybe. We could omit the "free" since technically we would have to add it to the spectral part too, and thats too much bloat i feel like.